spec/integration/rendering_spec.rb

```diff
@@ -9,20 +9,21 @@
     expected = <<~DESC
       Commands:
         foo assets [SUBCOMMAND]
-        foo callbacks DIR                      # Command with callbacks
-        foo console                            # Starts Foo console
+        foo callbacks DIR                                          # Command with callbacks
+        foo console                                                # Starts Foo console
         foo db [SUBCOMMAND]
         foo destroy [SUBCOMMAND]
-        foo exec TASK [DIRS]                   # Execute a task
+        foo exec TASK [DIRS]                                       # Execute a task
         foo generate [SUBCOMMAND]
         foo greeting [RESPONSE]
-        foo hello                              # Print a greeting
-        foo new PROJECT                        # Generate a new Foo project
-        foo routes                             # Print routes
-        foo server                             # Start Foo server (only for development)
+        foo hello                                                  # Print a greeting
+        foo new PROJECT                                            # Generate a new Foo project
+        foo root-command [ARGUMENT|SUBCOMMAND]                     # Root command with arguments and subcommands
+        foo routes                                                 # Print routes
+        foo server                                                 # Start Foo server (only for development)
         foo sub [SUBCOMMAND]
         foo variadic [SUBCOMMAND]
-        foo version                            # Print Foo version
+        foo version                                                # Print Foo version
```

Comment:
@jodosha this tabs makes me crazy, in a specs, can't we remove it (replace with rjust/ljust)? :(

Reply:
I'm for removing them altogether to be honest. It's just a waste of space and it can easily make certain lines not fit and wrap or not be visible in a terminal (thus require horizontal scrolling). I understand the aesthetic aspect of such alignment but it doesn't seem to be worth the effort.

Reply:
@IvanShamatov Please proceed.